frontend/web/components/modals/CreateFlag.js

```diff
@@ -184,6 +186,13 @@ const CreateFlag = class extends Component {
     return Promise.resolve(true)
   }
 
+  getReleasePipelineId = () => {
+    const releasePipeline = this.state.releasePipelines?.find((pipeline) =>
```

Comment:
L255 It seems it's missing to set the pipelines state:
```
        this.setState({
          hasPublishedReleasePipelines,
          releasePipelines: pipelines,
        })
```

not working for me without this change 🤔